apps/pom.xml

```diff
@@ -119,7 +119,7 @@
 		<!-- Dependencies required for connecting to a gRPC API -->
 		<grpc_java.version>1.49.2</grpc_java.version>
 		<protoc.version>3.21.2</protoc.version>
-		<protobuf.version>3.21.7</protobuf.version>
+		<protobuf.version>3.21.8</protobuf.version>
 
 		<!-- The default DB that will be used in integration tests. -->
 		<!-- Note: See gov.cms.bfd.model.rif.schema.DatabaseTestHelper for details
```

Comment:
Hapi FHIR was picked up. I looked at our runbook for upgrading dependencies and saw that it was on the list of  dependencies that we should be careful upgrading. Should I just ignore what's in the runbook and ugprade?

Reply:
That runbook is for resolving the SNYK auto-generated PRs. For this exercise we still want to be careful but try to get everything to latest mutually compatible versions. I don't expect trouble from this upgrade since it is a minor version. Were there any other libs that were held for the same reason?

Reply:
ok, let's bring that one in too provided that all of the regression passes.

Reply:
OK brought both in